src/hooks/useSelectionState/useSelectionState.stories.tsx

```diff
@@ -106,3 +106,52 @@ export const ExpandableTable: React.FunctionComponent = () => {
     </Table>
   );
 };
+
+export const ExternalState: React.FunctionComponent = () => {
+  // import { Checkbox } from '@patternfly/react-core';
+
+  interface IFruit {
+    name: string;
+    // Anything else, this can come straight from API data
+  }
+
+  // Somewhere else in your app, perhaps:
```

Comment:
Not sure about the usefulness of this comment.

Reply:
Fair point.